### PR TITLE
#160538240 bg fix place order id

### PR DESCRIPTION
**Pull request by marcdomain**
### What does this PR do?
- Fix new-order id bugs, destructure methods, update README.md
### Description of Task to be completed
- Update order id variable in placeOrder controller
- Destructure validations and controller methods in route file
- Update README.md
### How should this be manually tested?
- Use postman to access the endpoints via https://marcus-fast-food-fast.herokuapp.com/api/v1/
- Test all the endpoints using their respective endpoints
### What are the relevant pivotal tracker stories?
- #160538240

### CODE REVIEW

File: server/routes/orderRoutes.js [hunk 1]
```diff
@@ -2,12 +2,18 @@ import express from 'express';
 import ordersController from '../controllers/ordersController';
 import validators from '../middlewares/orderInputValidations';
 
+const {
```
Comment (proxiex) [inline server/routes/orderRoutes.js]: It's good your are deconstructing. But you can easily do this at line 2

File: server/routes/orderRoutes.js [hunk 2]
```diff
@@ -2,12 +2,18 @@ import express from 'express';
...
+  placeOrder, getAllOrders, fetchSpecificOrder, updateOrder, deleteOrder
+} = ordersController;
+
+const { placeOrderValidator, fetchSpecificOrderValidator } = validators;
```
Comment (proxiex) [inline server/routes/orderRoutes.js]: It's good your are deconstructing. But you can easily do this at line 3

**Review by proxiex — APPROVED** [review]

File: server/routes/orderRoutes.js (hunk 1 above)
Comment (marcdomain) [reply server/routes/orderRoutes.js]: Thanks for the observation. I've made the adjustment and it's more tidy.

File: server/routes/orderRoutes.js (hunk 2 above)
Comment (marcdomain) [reply server/routes/orderRoutes.js]: Thanks for the observation. I've made the adjustment and it's more tidy.

**Pull request merged**